.github/workflows/rails-integration-tests.yml

```diff
@@ -16,8 +16,9 @@ jobs:
       # Get latest Rails versions for 6.x.x and 7.x.x
       - id: compute-outputs
         name: Compute outputs
+        # fetches current Rails versions numbers > 5 and not 'beta'
         run: |
-          rails_versions=$(curl https://rubygems.org/api/v1/versions/rails.json | jq 'group_by(.number[:1])[] | (.[0].number) | select(.[:1]|tonumber > 5)' | jq -s -c)
+          rails_versions=$(curl https://rubygems.org/api/v1/versions/rails.json | jq '[.[] | select(.number | test("beta") | not)] | group_by(.number[:1])[] | (.[0].number) | select(.[:1]|tonumber > 5)' | jq -s -c)
```

Comment:
I don't know the jq syntax well enough to review this line

Reply:
Honestly, me too, I would prefer something less [dynamic but more obvious](https://github.com/dxw/mail-notify/pull/158#discussion_r1700134381), but happy to trust in this having seen it work.